spec/io_spec.rb

```diff
@@ -26,7 +26,7 @@
 #
 require "tmpdir" # Used to avoid cluttering the repository.
 require_relative 'spec_helper'
-require_relative "../lib/nmatrix"
+require "./lib/nmatrix"
```

Comment:
I made these changes after looking at git diff. I am sure that gem will work properly when installed.

How do I correct remaining travis tests?